src/file/reader.rs

```diff
@@ -21,7 +21,11 @@ use std::rc::Rc;
 
 use basic::{Type, Compression, Encoding};
 use errors::{Result, ParquetError};
-use file::metadata::{RowGroupMetaData, FileMetaData, ParquetMetaData};
+use file::metadata::{
```

Comment:
How do you feel about using `use file::metadata::*` here?

Reply:
Yes. Makes sense. I'll replace with star-import.